components/Form/AccordionItem/index.module.scss

```diff
@@ -0,0 +1,5 @@
+@import '../../../pages/stylesheets/vars.scss';
+
+.lbh-accordion__section-header {
+  color: $hackney-green !important;
```

Comment:
Is there a way to do it without important? 🤔

Reply:
I don't think it matters at all as the css is scoped

Reply:
Fair